### PR TITLE
“#8Task. Ilya Maylychko

### PR DESCRIPTION
**Pull request by juggler39**
https://travis-ci.org/github/juggler39/js-assignments/builds/708880848

### CODE REVIEW

File: task/08-objects-tasks.js [hunk 1]
```diff
@@ -107,36 +109,69 @@ function fromJSON(proto, json) {
  */
 
 const cssSelectorBuilder = {
```
Comment (AlexMokin) [inline task/08-objects-tasks.js:171]: Please review the task carefully it says
**Provided cssSelectorBuilder should be used as facade only to create your own classes**

Please make sure you use cssSelectorBuilder  only as a facade

**Comment by AlexMokin** [comment]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by juggler39** [comment]
I don't see any comments. Could you explain me please, what is wrong with this task.

**Review by AlexMokin — COMMENTED** [review]
See below
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

File: task/08-objects-tasks.js (hunk 1 above)
Comment (juggler39) [reply task/08-objects-tasks.js:171]: I've rewritten the class using facade pattern.

**Comment by IlyaMokin** [comment]
Closing as accepted! Thanks!

**Pull request closed without merging**